NEWS

```diff
@@ -161,6 +161,16 @@ New Functionality
 - Add logging metrics for streams (``zeek-log-stream-writes``) and writers
   (``zeek-log-writer-writes-total``).
 
+- Add networking metrics via the telemetry framework. These are enabled
+  when the ``misc/stats`` package is loaded and updated every 15 seconds
```

Comment:
Total nits: I suggest "script" here instead of "package", and perhaps a reference to `Stats::Info` for context on where they come from.